app/components/candidate_interface/course_choices_review_component.html.erb

```diff
@@ -62,9 +62,9 @@
         </li>
       </ul>
     <% elsif application_choice.study_mode_full? %>
-      <p class='app-review-warning__primary-message'><%= application_choice.study_mode_full_error %>.</p>
-      <p class='govuk-body'>You can:</p>
-      <ul class='govuk-list govuk-list--bullet'>
+      <p class="govuk-body app-inset-text__title"><%= application_choice.study_mode_full_error %>.</p>
```

Comment:
Potentially we could add the relevant sizing / spacing css to `app-inset-text__title` to avoid the need to also add `govuk-body`? But it's probably fine as-is. I'm never sure about when to nest components...

Reply:
Good point. Have updated `.app-inset-text__title` styles, and removed `govuk-body` from these paragraphs.